java/java.disco/src/org/netbeans/modules/java/disco/SelectPackagePanel.java

```diff
@@ -177,6 +194,11 @@ protected void updateData(Distribution distribution, Integer featureVersion, Lat
                 .execute();
     }
 
+    @Override
+    protected void updateDistributions(List<Distribution> distros) {
+        super.updateDistributions(distros); //To change body of generated methods, choose Tools | Templates.
```

Comment:
someone hasn't updated his code templates :P

Reply:
Oh $%$*&! I'll tidy this up too.